Python/pdf_reorder/pdf_reorder_.py

```diff
@@ -0,0 +1,50 @@
+
+from pdfrw import PdfReader, PdfWriter
+
+# The function input_and_parse gathers the inputs and parses and then sorts them in
+# the order required by reorder(path,dic) function
+#It takes one argument which is page length of pdf 'n'

```

Comment:
Use Multi-Line Strings instead of multiple Single-Line Comments.

Reply:
Okay done that